components/v1/PortofolioPage/PortofolioPage.tsx

```diff
@@ -122,16 +124,26 @@ const PortofolioPage: FunctionComponent<PortofolioPageProps> = ({}) => {
                                         <tbody>
                                             {/* TODO(Matthew): Use map if there are more than 1 Leveraged token */}
                                             {isHavePortofolio ? (
-                                                <tr className="text-right text-sm font-semibold ">
+                                                <tr className="text-right text-sm font-semibold">
                                                     <td className="flex items-center space-x-4 text-left">
                                                         <img className="h-[40px] w-[40px]" src={metadata.logo} alt={metadata.title} />
-                                                        <p className="text-gray-light-12 dark:text-gray-dark-12">{metadata.title}</p>
+                                                        <div className="flex flex-col space-y-[8px]">
+                                                            <p className="text-gray-light-12 dark:text-gray-dark-12">{metadata.title}</p>
+                                                            <p className="text-gray-light-10 dark:text-gray-dark-10 lg:hidden">
+                                                                {formattedEthriseBalance.toFixed(2)} {metadata.title}
```

Comment:
@math-marcellino Please use `tokenBalanceFormmater` for this

Reply:
Okay sir, but if we use `tokenBalanceFormatter`, there will be more than 2 decimal behind the coma, which is different from the design provided in figma, is that okay?
![image](https://user-images.githubusercontent.com/81855912/159872525-f5aff6ce-eed8-42af-b5b8-ceaa9d0ba7ec.png)

Reply:
@math-marcellino it's okay. Coz we want to make display amount consistent across the app. May be we want to remove the symbol on the `Amount` column? 

cc @ayikfour

Reply:
Okay noted sir @pyk . Btw I also agree to remove the symbol from `Amount` column, because it is too long and it is already displayed on the `Token` column.

cc @ayikfour